test/functional/interface_rest.py

```diff
@@ -91,25 +90,32 @@ def run_test(self):
 
         txid = self.nodes[0].sendtoaddress(self.nodes[1].getnewaddress(), 0.1)
         self.sync_all()
```

Comment:
@jnewbery, mostly an FYI, but open to feedback.. 
as per our conversation I've been trying to improve the variable propagation delay. The solution Ive been looking at is to use the `[create / sign / send]rawtransaction` RPCs to directly broadcast the tx to both nodes.
My attempt lead me running into the absurdly-high-fee error from `sendrawtransaction`. And the simple fix of passing in the “allow” boolean leads to an insufficient-funds error when trying to `sendtoaddress` later.
I decided to just put up these changes since they don’t degrade the performance, but I’ll probably keep digging because I’m curious to understand where fee is getting set & what Im doing wrong.

Reply:
These changes are good on their own and achieve the goal of removing `-txindex`.

To manually propagate the transaction, leave the `sendtoaddress()` in place, but then fetch the hex transaction using `getrawtransaction(txid)` (which works since the tx is still in the mempool), and then submit that transaction to the other nodes using `sendrawtransaction(hex_tx)`